docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
+----------------------------------------------------------------------
+
+endif::[]
+
+*win:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {ls} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the {ls} {logstash_version} Windows zip file from the
+https://www.elastic.co/downloads/logstash[{ls} download] page.
+
+. Extract the contents of the zip file to a directory on your computer, for
+example, `C:\Program Files`.
+
+endif::[]
+
+For other operating systems, go to the
+https://www.elastic.co/downloads/logstash[{ls} download] page.
+
+To learn more about installing, configuring, and running {ls}, read the
+{logstash-ref}/index.html[{ls} Reference].
+
+// REVIEWERS: I've removed the info about updating the beats input plugin for
+// Logstah because this tutorial tells people to download the latest version
+// and in most cases the input plugin version will be recent...so I'm not sure
+// we need to cover that here.
+
+[float]
+==== Configure {metricbeat} to send events to {ls}
+
+{metricbeat} sends events to {es} by default. To send events to {ls}, modify the
+{metricbeat} configuration file, `metricbeat.yml`. You'll find this file under
+the {metricbeat} install directory, or `/etc/metricbeat` for rpm and deb.
+
+Disable the `output.elasticsearch` section by commenting it out, then enable
+the `output.logstash` section by uncommenting it: 
+
+[source,yaml]
+----
+#-------------------------- Elasticsearch output ------------------------------
+#output.elasticsearch:
+  # Array of hosts to connect to.
+  #hosts: ["localhost:9200"]
+.
+.
+.
+#----------------------------- Logstash output --------------------------------
+output.logstash:
+  # The Logstash hosts
+  hosts: ["localhost:5044"]
+----
+
+
+Save the file, then restart {metricbeat} to apply the changes.
+
+
+[float]
+[[logstash-setup]]
+==== Configure {ls} to read from the {beats} input
+
+{ls} provides {logstash-ref}/input-plugins.html[input plugins] for reading from
+a variety of inputs. In this tutorial, you create a {ls} pipeline configuration
+that reads from the {beats} input and sends events to the {es} output.
+
+To configure {ls}:
+
+. Create a new {ls} pipeline configuration file called `logstash.conf` that
+contains:
++
+--
+* An input stage that configures {ls} to listen on port 5044 for incoming {beats}
+connections.
+* An output stage that indexes events into {es}. The output stage also
+configures {ls} to write to the {metricbeat} index.
+--
++
+For example:
++
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+# The filter part of this file is commented out to indicate that it
+# is optional.
+# filter {
+#
+# }
+
+output {
+  elasticsearch {
+    hosts => "localhost:9200"
+    manage_template => false
+    index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}" <1>
+  }
+}
+----
++
+This configuration allows you to route events to {ls}, where you have full
```

Comment:
The phrasing here leads me to believe that this configuration is the _source_ of the events,
but in reality it is the _recipient_. Perhaps something like:


~~~
When {ls} is started with this pipeline configuration, you will be able to route events
from {beats} to {elasticsearch} through {ls}, where you have full access to the {ls}
capabilities for collecting, enriching, and transforming data.
~~~